plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/handler/StartBuildHandler.java

```diff
@@ -52,29 +49,11 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 	}
 	
 	public IResource getBuildableResource(ISelection selection, String source) {
-		IResource buildAble = null;
-		IAdaptable adaptable = UIUtils.getFirstElement(selection, IAdaptable.class);
-		if(adaptable != null) {
-			buildAble = adaptable.getAdapter(IBuildConfig.class);
-			if (buildAble == null) {
```

Comment:
Looks like you lost the ability to initiate a build from a buildconfig which is probably our first choice.  A user would mostly start a new build then clone a previous one